### DISCUSSION TITLE
Connecting scanner/radio

### DISCUSSION THREAD

**Discussion by TPSSCW** [chuot/rdio-scanner#318] [Q&A]
I have 5 radios and 2 scanners that are all connected to a different sound card. Is there a way to connect them to rdio-scanner? I planned on using dir watch but haven't found a good vox program that will let me set up multiple vox recordings simultaneously with each saving to a multiple folder. I tried using api via voxcall but that didn't seem to work unless I didn't have something set right

**Comment by chuot** [comment]
You can use [voxcall](https://github.com/aaknitt/voxcall) which should be able the create files from all of your soundcards. Those files will be split by silences. voxcall has to be configured with the API key you will create within Rdio Scanner. The url to use for the API is the same as the one you're using to access the integrated webapp, which is usually on port 3000. As for the system ID and talkgroup ID, you put whatever value you want, Rdio Scanner scanner will create the systems and the talkgroups for you since the autopopulate option is activated by default. Then you can use the Rdio Scanner admin interface to change the name for the system, and for the talkgroup. Give it a try.

I never tried voxcall, but it works.

**Comment by TPSSCW** [comment]
I put the api key in but it still didn't work. I also can not connect to 
rdio-scanner when I'm not at home on my own network. I even tried the 
downstream feature using another pc on my network with no success, kept 
saying invalid api key

**Comment by chuot** [comment]
Have you defined the API key in Rdio Scanner ?

As for connecting from outside of your home, I've answer that in the other discussion.

**Comment by TPSSCW** [comment]
Yes, I want to set up a downstream from another location to my home location like you had helped another person with but figured I would try it on 2 pc on my own network before going out to the other location and trying it. I generated the key on my main instance of rdio-scanner and pasted it into the downstream instance and key an invalid api key notice  Sent from Mail for Windows From: Chrystian HuotSent: Tuesday, January 24, 2023 7:43 AMTo: chuot/rdio-scannerCc: TPSSCW; AuthorSubject: Re: [chuot/rdio-scanner] Connecting scanner/radio (Discussion #318) Have you defined the API key in Rdio Scanner ?As for connecting from outside of your home, I've answer that in the other discussion.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
